modules/tensor_mechanics/src/kernels/ADStressDivergenceRSphericalTensors.C

```diff
@@ -0,0 +1,55 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "ADStressDivergenceRSphericalTensors.h"
+#include "ElasticityTensorTools.h"
+#include "FEProblem.h"
+#include "MooseMesh.h"
+
+registerADMooseObject("TensorMechanicsApp", ADStressDivergenceRSphericalTensors);
+
+defineADValidParams(
+    ADStressDivergenceRSphericalTensors,
+    ADStressDivergenceTensors,
+    params.addClassDescription(
+        "Calculate stress divergence for a spherically symmetric 1D problem in polar coordinates.");
+    params.addRequiredParam<unsigned int>(
+        "component",
+        "An integer corresponding to the direction the variable this kernel acts in. (0 "
+        "for x, 1 for y, 2 for z; note in this kernel x refers to the radial "
+        "displacement.)");
+    params.set<bool>("use_displaced_mesh") = true;
```

Comment:
Another duplicate. do we want it here?

Reply:
Not a duplicate, because the base class sets it to false (which is confusing a.f.). I'm removing this.